lib/OpenQA/Schema/Result/JobTemplates.pm

```diff
@@ -70,12 +70,8 @@ Returns a hash with the assigned settings.
 sub settings_hash {
     my ($self) = @_;
 
-    my $settings = $self->settings;
-    my %settings_hash;
-    while (my $setting = $settings->next) {
-        $settings_hash{$setting->key} = $setting->value;
-    }
-    return \%settings_hash;
+    my @settings = sort { $a->key cmp $b->key } $self->settings;
+    return [map { {key => $_, value => $_->value} } @settings] if (@settings);
```

Comment:
The parentheses there do nothing, you can just write `if @settings;`. I don't mind this too much, just wanted to make sure you know.